src/main/java/geekbrains/java2/ht2/Main.java

```diff
@@ -0,0 +1,42 @@
+package geekbrains.java2.ht2;
+
+public class Main {
+
+	/**
```

Comment:
\[Checkstyle\] ERROR: File contains tab characters \(this is the first instance\)\.